test/images/image-util.sh

```diff
@@ -21,15 +21,15 @@ set -o pipefail
 TASK=$1
 IMAGE=$2
 
-KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE}")/../.." && pwd -P)"
+KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
```

Comment:
I hope the addition of `[0]` doesn't change behavior...?

Reply:
Thanks for your feedback. I am sorry for the delay. 
But why do you think so? 
I don't think the addition of `[0]` will change the behavior of this script, because expanding an array without an index only gives the first element.

Here's what shellcheck finds:
```
In test/images/image-util.sh line 24:
KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE}")/../.." && pwd -P)"
                            ^-- SC2128: Expanding an array without an index only gives the first element.
```

Reply:
Thanks for clarifying, LGTM